packages/ember/typescript/src/package-json.js

```diff
@@ -7,7 +7,7 @@ export async function adjustAppScripts() {
 import { canUseBuiltInTypes, isEmberDataPresent } from './queries.js';
 
 export async function adjustAddonScripts() {
-  throw new Error(`Addons are not yet supported. PR's welcome!`);
```

Comment:
I'm not 100% sure if this is all that needs to happen here, I can change it to a warning instead so people can double check?

Reply:
yeah, the way types are built in ember-cli-typescript is _bonkers_, and folks may want that bonkers behavior added to their scripts here